drivers/hmis_supplemental/app/views/hmis_supplemental/data_sets/_form.haml

```diff
@@ -0,0 +1,22 @@
+= simple_form_for data_set, as: :data_set do |f|
+  - if data_set.errors.any?
+    .alert.alert-danger{style: 'display: block;'}
+      %p Failed to save
+      %ul.error_list
+        - data_set.errors.full_messages.uniq.each do |msg|
+          %li= msg
+
+  .form-inputs
+    = f.input :name, required: true
+    = f.input :owner_type, collection: ['enrollment', 'client'], required: true
+    = f.input :data_source_id, collection: data_sources.order(:name).map {|d| [d.name, d.id]}, required:true
+    = f.input :field_config, as: :text, label: 'Field Configuration JSON', hint: 'example: `[{"key":"widget","type":"float","label":"The Widget","multiValued":false}]`', input_html: {rows: 10}
+    = f.fields_for(:remote_credential) do |sf|
+      %fieldset
+        %legend S3 Configuration
+        = sf.input :bucket, as: :string, label: 'S3 bucket'
```

Comment:
Pretty sure we need `region` as well, see `app/views/hmis_import_configs/_form.haml` for an example.